inference-engine/src/gna_plugin/layers/gna_layer_info.hpp

```diff
@@ -90,6 +90,27 @@ class LayerInfo {
         static InferenceEngine::details::caseless_set<std::string> layersWithConstrains = {"memory", "convolution"};
         return layersWithConstrains.find(name) != layersWithConstrains.end();
     }
+    size_t getBatchSize() const {
```

Comment:
The very confusing function name which is matched with API function.

Reply:
The function was renamed.